pyproject.toml

```diff
@@ -1,45 +1,76 @@
-[tool.poetry]
+[project]
+authors = [{name = "Alexander Fischer", email = "alexander-fischer1801@t-online.de"}]
+description = "Fast high dimensional fixed effect estimation following syntax of the fixest R package."
 name = "pyfixest"
+requires-python = ">= 3.9"
 version = "0.24.1"
+dependencies = ["lets-plot>=4.0.1"]
```

Comment:
Don't we need the runtime dependencies here? See https://github.com/ericmjl/llamabot/blob/main/pyproject.toml (from the author of https://ericmjl.github.io/blog/2024/8/16/its-time-to-try-out-pixi/). I am not 100% sure about it.

Reply:
`lets-plot` cannot be installed from `conda`, so I added it via `pixi add lets_plot --pypi`, which placed it there in the toml. I think (but am not sure) that this is where `pixi` places all non-conda deps?